doc/over_react_redux_documentation.md

```diff
@@ -0,0 +1,320 @@
+# OverReact Redux
+> A Dart React Redux wrapper for targeted state updates.
```

Comment:
#nit reword this to reduce ambiguity around the grouping of the words "Dart React Redux"
```suggestion
> A Dart wrapper for React Redux, providing targeted state updates.
```